modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -121,40 +125,64 @@ image::manage-security/configureLDAPencryptionPullDownMenu.png[,200,align=left]
 +
 The options are *None* (to connect without encryption &#8212; this is insecure, and therefore is _not_ recommended), *TLS* (to connect to a TLS-encrypted port), and *StartTLSExtension* (to upgrade an existing connection).
 
-* *Certificate Validation*. Whether to validate Couchbase Server with the server certificate.
-This set of radio-buttons is enabled only if *TLS* or *StartTLSExtension* has been selected from the *Encryption* pull-down menu.
+* *Certificate*. Whether to validate Couchbase Server by presenting the root certificate of the LDAP server.
```

Comment:
In my opinion, word 'with' was better in this case, because this cert is used by Couchbase Server. If you think it's still clear after the change, then I don't mind the change.

Reply:
Yes, I've modified this.